### PR TITLE
[RHPAM-3654] - update Kogito version for product release 7.11.1.GA

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Review by vaibhavjainwiz — COMMENTED** [review]
LGTM

File: image.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ modules:
 
 envs:
   - name: "KOGITO_VERSION"
-    value: "1.5.1-SNAPSHOT"
+    value: "1.5.0.redhat-00006"
```
Comment (radtriste) [inline image.yaml:21]: This should be set by prod pipelines. not directly here I would say ...

File: tests/features/rhpam-kogito-builder-jvm.feature [hunk 2]
```diff
@@ -44,7 +44,7 @@ Feature: rhpam-kogito-builder-rhel8 feature.
       | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
 
   Scenario: Verify that the Kogito Maven archetype is generating the project and compiling it correctly with custom group id, archetype & version
-    Given s2i build /tmp/kogito-examples from dmn-example using master and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
+    Given s2i build /tmp/kogito-examples from dmn-example using 1.5. and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```
Comment (radtriste) [inline tests/features/rhpam-kogito-builder-jvm.feature]:
```suggestion
    Given s2i build /tmp/kogito-examples from dmn-example using 1.5.x and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```

File: tests/features/rhpam-kogito-builder-jvm.feature [hunk 3]
```diff
@@ -67,7 +67,7 @@ Feature: rhpam-kogito-builder-rhel8 feature.
 #### SpringBoot Scenarios
 
   Scenario: Verify that the Kogito Maven archetype is generating the project and compiling it correctly when runtime is springboot
-    Given s2i build /tmp/kogito-examples from dmn-example using master and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
+    Given s2i build /tmp/kogito-examples from dmn-example using 1.5. and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```
Comment (radtriste) [inline tests/features/rhpam-kogito-builder-jvm.feature]:
```suggestion
    Given s2i build /tmp/kogito-examples from dmn-example using 1.5.x and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```

**Review by radtriste — CHANGES_REQUESTED** [review]

File: image.yaml (hunk 1 above)
Comment (spolti) [reply image.yaml:21]: discussed on gchat, let's update it here fort his release, later we can define a better way update it, for prod builds and upstream.

**Pull request merged**